### PR TITLE
Replace deprecated `TimedJob` class

### PR DESCRIPTION
**Pull request by gemini-git**
The used class `TimedJob` is deprecated. I have replaced it with the proposed class.

### CODE REVIEW

File: lib/Cron/AutoExport.php [hunk 1]
```diff
@@ -9,24 +9,35 @@
...
 	 */
 	private $projectService;
 
-	public function __construct(ProjectService $projectService) {
+	/**
+	 * @param ITimeFactory $time
+	 * @param ProjectService $projectService
+	 */
+	public function __construct(ITimeFactory $time, ProjectService $projectService)
+	{
```
Comment (eneiluj) [inline lib/Cron/AutoExport.php:27]: Could you keep the function opening curly bracket on the same line?

```suggestion
	public function __construct(ITimeFactory $time, ProjectService $projectService) {
```

**Review by eneiluj — CHANGES_REQUESTED** [review]
:+1: Thanks a lot

File: lib/Cron/AutoExport.php (hunk 1 above)
Comment (gemini-git) [reply lib/Cron/AutoExport.php]: I changed it and also changed the code style settings in my ide to automatically use the "same line" option.

**Review by eneiluj — APPROVED** [review]
:+1: Thanks!

**Pull request merged**